mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -314,109 +328,73 @@ export const Columns = (props: ColumnsProps) => {
         );
       },
     }),
-    columnHelper.accessor("studentComments", {
-      header: "Student comments",
+    columnHelper.accessor("comments", {
+      header: "Comments",
 
       /**
-       * Cell with StudentComments items
+       * Cell with Comments items
        */
       cell: ({row}) => {
+        const userUuid = sessionStorage.getItem("user") ?? "";
```

Comment:
what does this string do in the sessionStorage? 
1 - don't use magic value
2 - get rid of session storage

Reply:
As I told I need sessionStorage do I can't get rid it right now.

Reply:
get rid of session storage

Reply:
ADded context instead of sessionStorage